example-app/src/config.ts

```diff
@@ -1,3 +1,8 @@
 export const GRAPHQL_URI = process.env.REACT_APP_GRAPHQL_URI ?? "/graphql";
 export const LOGIN_URI = process.env.REACT_APP_LOGIN_URI ?? "/login";
 export const LOGOUT_URI = process.env.REACT_APP_LOGOUT_URI ?? "/logout";
+export const REQUEST_CREDENTIALS = process.env.REACT_APP_REQUEST_SAME_ORIGIN
+  ? process.env.REACT_APP_REQUEST_SAME_ORIGIN === "true"
+    ? "same-origin"
+    : "include"
+  : "same-origin";
```

Comment:
We should not have complex logic here. Please move the logic to where the environment variable is used. Also, please do not use nested ternary operator, it is hard to read.

```suggestion
export const REQUEST_SAME_ORIGIN = (process.env.REACT_APP_REQUEST_SAME_ORIGIN === "true");
```